<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/icon.ico</normaloff>img/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:rgb(47, 47, 47);
color:rgb(200, 200, 200)
}
QTabWidget::pane {
	border: 1px solid rgb(128, 128, 128);
	border-radius: 3px;
}
QTabBar::tab 
{
    background: #48555E;
    color: white;   
	padding:5px;
    border: 1px solid #000;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;
}
QTabBar::tab:selected, 
QTabBar::tab:hover 
{
    border-top-color: #1D2A32;
    border-color: #40494E;
    color: black;
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #C1D8E8, stop: 1 #F0F5F8); 
}
QLineEdit, QComboBox, QTextEdit{
	background-color: rgb(85,85,85);
	border-color: rgb(0, 0, 0);
	border-width : 1px;
	border-style:inset;
	padding: 5px;
	border-radius:3px;
}
QPushButton{
	background:rgb(102, 102, 102);
	color:rgb(255, 255, 255);
	border-radius:3px;
	padding-top:5px;
	padding-bottom:5px;
	padding-right:10px;
	padding-left:10px;
	border-color: rgb(0, 0, 0);
	border-width : 1px;
	border-style:inset;
}
QPushButton:hover{
	background-color:rgb(128, 128, 128);
}
QPushButton:pressed{
	background-color:#A4A4A4;
	color:#000
}
QLabel{
	border:0px;
}
QCheckBox{
	border:0px;
}
QTextBrowser{
	color: rgb(230, 230, 230);
	padding:10px;
	background-image: url(template/img/logo.png);
	background-repeat: no-repeat;
  	background-attachment: fixed;
 	background-position: center;
}
QGroupBox {
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 0 5px;
}
QGroupBox::indicator {
    width: 13px;
    height: 13px;
}
QGroupBox#groupBoxNotifications::indicator:unchecked, QGroupBox#groupBoxNotifications::indicator:checked {
	image: url(template/img/notification.png);
}
QGroupBox#groupBoxSettings::indicator:unchecked, QGroupBox#groupBoxSettings::indicator:checked {
	image: url(template/img/settings.png);
}

QGroupBox#groupBoxBrowsers::indicator:unchecked, QGroupBox#groupBoxBrowsers::indicator:checked {
	image: url(template/img/browser.png);
}

QGroupBox#groupBoxTelegram::indicator:unchecked, QGroupBox#groupBoxTelegram::indicator:checked{
	image: url(template/img/telegram.png);
}

QGroupBox#groupBoxAccountSettings::indicator:unchecked, QGroupBox#groupBoxAccountSettings::indicator:checked{
	image: url(template/img/user.png);
}

QMenuBar {
    background-color: rgb(40,40,40);
}
QMenuBar::item {
    padding: 4px 8px;
    background: transparent;
    border-radius: 4px;
}

QMenuBar::item:selected { /* when selected using mouse or keyboard */
    background: rgb(60,60,60);
	color:rgb(200,200,200);
}

QMenuBar::item:pressed {
    background: #888888;
	color: #000;
}
QHeaderView::section, QHeaderView{
	background: #48555E;
	border: 0px;
	padding-left:3px;
}
QHeaderView::item{
	border: 0px;
}
QTableCornerButton::section {
    background: #48555E;
}
QTableWidget{
	border: 1px solid gray;
	border-radius:5px;
}
QTableWidget#tableWidgetPlanetList{
	border: 2px solid gray;
	border-radius:5px;
}
</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>630</width>
     <height>170</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset>
      <normaloff>img/security.png</normaloff>img/security.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Security</string>
    </attribute>
    <widget class="QCheckBox" name="checkBoxTLS13">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>50</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TLS 1.3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>185</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Security Level</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxHSTS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>HSTS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSaveSecurity">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>90</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAutomaticHTTPSRewrites">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Automatic HTTPS Rewrites</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxSecurityLevel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Essentially Off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Low</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medium</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>High</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Under Attack</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>15</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAlwaysUseHTTPS">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Allways use HTTPS</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>185</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SSL</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>15</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxSSL">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Flexible</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Full</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Strict</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="icon">
     <iconset>
      <normaloff>img/network.png</normaloff>img/network.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Network</string>
    </attribute>
    <widget class="QCheckBox" name="checkBoxHTTP3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>HTTP/3 (with QUIC)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxWebSockets">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>WebSockets</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxOnionRouting">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Onion Routing</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxIPGeolocation">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>50</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>IP Geolocation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSaveNetwork">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>90</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditID">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>100</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxMinTLS">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1.0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.3</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>15</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>185</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Minimum TLS Version</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset>
      <normaloff>img/cache.png</normaloff>img/cache.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Cache</string>
    </attribute>
    <widget class="QCheckBox" name="checkBoxDevelopmentMode">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Development Mode</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>10</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Browser Cache TTL</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAlwaysOnline">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Always Online</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxCacheLevel">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>10</y>
       <width>110</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Basic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Aggressive</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Simplified</string>
      </property>
     </item>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>3</width>
       <height>30</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxTTL">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>5 minutes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15 minutes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30 miniutes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>45 minutes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 hour</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2 hours</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3 hours</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4 hours</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8 hours</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16 hours</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 day</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 week</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 month</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1 year</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cache Level</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>15</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>10</y>
       <width>15</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSaveCache">
     <property name="geometry">
      <rect>
       <x>543</x>
       <y>90</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonClearCache">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>50</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Clelar Cache</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>666</width>
     <height>210</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
